freqtrade/rpc/api_server/ws/channel.py

```diff
@@ -160,6 +178,7 @@ async def on_disconnect(self, websocket: WebSocketType):
         with self._lock:
             channel = self.channels.get(websocket)
             if channel:
+                logger.info(f"Disconnecting channel {channel}")
```

Comment:
intentional on info? (not a problem necessarily - i guess a disconnect won't happen this often?)

Reply:
Yes this was intentional